### DISCUSSION TITLE
WDACSimulation

### DISCUSSION THREAD

**Discussion by andy4871** [HotCakeX/Harden-Windows-Security#224] [Q&A]
I am having hard time using invoke-wdacsimulation.
1. It’s failing when gci does not have permissions to the path (e.g. c:\programdata\Microsoft\Windows Defender Advanced Threat Protection Cache). I can ignore it with by providing -EA Continue common parameter, but not sure it’s there right choice. Would be great to have a parameter to skip files/dirs to which the process cannot have access (and a separate log with list of skipped files)
2. It stops with:
TerminatingError(Get-SignedFileCertificates): “Exception calling “Import” with “3” argument(s): “Cannot find the requested object.””

**Comment by HotCakeX** [comment]
Thanks for reporting it, I converted this to an issue, will fix it
